### ISSUE TITLE
No "Restart TS Server" when viewing diff editor

### ISSUE THREAD

**Issue by rebornix** [microsoft/vscode#201734]
* Working on a branch with changes, open a diff editor for a modified ts file
* Switch to another branch
* TS Server generates a lot of unexpected errors
* Now I want to restart the ts server to start fresh
* Open Command Palette, the command is not available

![image](https://github.com/microsoft/vscode/assets/876920/d2481550-c0a8-4b52-b475-36256a5dc34b)

**Comment by eleanorjboyd** [comment]
So I should be creating a TS server or is this something with the editor?

**Comment by eleanorjboyd** [comment]
I am unable to get the `TypeScript: Restart TS Server` to appear if I am in the left side of the diff view.

<img width="1762" alt="image" src="https://github.com/microsoft/vscode/assets/26030610/02d1402c-083f-452b-b8a8-b60fc9bb231f">

**Comment by mjbvz** [comment]
Yes this is still disabled for `git:` files. Will take a look in February
